PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -9,8 +9,8 @@
 local PlayersService = game:GetService('Players')
 local UserInputService = game:GetService('UserInputService')
 
---NoVR will be false if the platform does not have the IsVREnabled property, because indexing it will throw an error.
-local success = pcall(function() local _ = UserInputService.IsVREnabled end)
+--NoVR will be false if the platform does not have the VREnabled property, because indexing it will throw an error.
+local success = pcall(function() local _ = UserInputService.VREnabled end)
 local NoVR = not success
```

Comment:
The naming of the 'NoVR' variable is very confusing because it doesn't mean VREnabled is false it just means that the API has not shipped yet. Something more verbose like 'VRApiExists' would make a lot more sense and avoid potential future bugs.